### ISSUE TITLE
Please add `git` executable for releases from git repository

### ISSUE THREAD

**Issue by rkoster** [cloudfoundry/bosh-deployment-resource#6]
When trying to use the following runtime config:
```yaml
releases:
  - name: dns
    url: git+https://github.com/cppforlife/dns-release
    version: 4
```

I get:
```
Could not deploy: Processing release 'dns/4': Cloning git repo: Starting command 'git clone https://github.com/cppforlife/dns-release --depth 1 /root/.bosh/tmp/bosh-upload-release-git-clone791798605': exec: "git": executable file not found in $PATH
```

**Comment by wendorf** [comment]
Thanks for the report! I just released v2.4.0 of the resource (tagged `latest` on Docker Hub, so you should just get it) that fixes this issue.

**Comment by rkoster** [comment]
thanks!
